### DISCUSSION TITLE
Windows cmd randomly disappears when resizing

### DISCUSSION THREAD

**Discussion by XCarCedo** [Textualize/textual#2936] [Q&A] [answered]
So i'm a little new to textual when i try to run the examples by python -m textual cmd randomly disappears sometimes when i try to resize it but it still remains in the task manager and have use of cpu, though this problem doesn't seem to happened at all in Windows Terminal.
https://github.com/Textualize/textual/assets/69898738/3aee48ca-903c-46cb-a27c-f8d74593ac11

i'm using textual version 0.29.0 on windows 10

**Comment by davep** [comment]
Testing this with `conhost.exe` on Windows 11, I'm unable to recreate this. It's hard to say what could be causing this but on the surface this would seem to be an issue with conhost on Windows 10. Note that we very much encourage the use of Windows Terminal for many different reasons.

**Comment by XCarCedo** [comment]
I'm pretty sure i didn't had this issue on my previous windows installation, and i forgot to mention that i'm using atlas os a custom edition of windows 10 and the version is 2009 (build 19042.631) so maybe that's where the problem is coming from? idk i'm not sure and i guess you guys doesn't provide support for these custom editions of win 10 (and it's not the only time i'm having issues with windows cmd around 4 5 ago i guess i've mis configured something that made cmd history much shorter but anywhere i've searched i just couldn't find anything to reset, reinstall or anything like that)

**Reply by willmcgugan** [reply, marked as answer]
Yeah, afraid it doesn't look like a Textual issue. Not even sure how we could cause this if we tried!
